pkgs/development/tools/misc/gpuvis/default.nix

```diff
@@ -0,0 +1,44 @@
+{ fetchFromGitHub
+, freetype
+, gtk3
+, lib
+, meson
+, ninja
+, pkg-config
+, SDL2
+, stdenv
+, wrapGAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gpuvis";
+  version = "20210108";
+
+  src = fetchFromGitHub {
+    owner = "mikesart";
+    repo = pname;
+    rev = "edf388d443287cc3b628a2574cacce93d235dce0";
+    sha256 = "1db5vxzm5a31av2v58vq80hc8wfaigpv9f0dp9q9k6rara84m3qq";
+  };
+
+  # patch dlopen path for gtk3
+  # python2 is wrongly added in the meson file, upstream PR: https://github.com/mikesart/gpuvis/pull/62
+  postPatch = ''
+    substituteInPlace src/hook_gtk3.h \
+      --replace "libgtk-3.so" "${lib.getLib gtk3}/lib/libgtk-3.so"
+    substituteInPlace meson.build \
+      --replace "prog_python2 = find_program('python2')" ""
```

Comment:
You're probably aware but a recent commit removed this so next time you version bump you can remove this :)

https://github.com/mikesart/gpuvis/commit/216f7d810e182a89fd96ab9fad2a5c2b1e425ea9

Reply:
Yeah, since I sent the PR :grin: I'll update this in a bit.

Reply:
Ah lol, I thought I checked you weren't one of the 2 committers but i must have tunnel visioned on the second